library/python/envoy_mobile/executor.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <functional>
+
+
+class Executor {
```

Comment:
I don't know that we should necessarily have this, since it's basically where we'd want to plug in native python dispatch. I mean, it _is_ a flexible and pretty agnostic interface, but does it really align with python based dispatch mechanisms?

Reply:
it's necessary to have a type defined in C++ for the C++ side of the lib to interact with it, e.g. to call into it from `on_XXX`. in practice this is going to work like:

[executor_base.h](https://github.com/crockeo/py-envoy-mobile/blob/main/py_envoy_mobile/wrapper/executor_base.h) being implemented by something akin to [asyncio_executor.py](https://github.com/crockeo/py-envoy-mobile/blob/main/py_envoy_mobile/asyncio_executor.py) or [gevent_executor.py](https://github.com/crockeo/py-envoy-mobile/blob/main/py_envoy_mobile/gevent_executor.py) inside of Python

Reply:
Okay, cool - if it's directly supported with gevent/asyncio (and the "right" way to integrate) then that's totally fine.